### PR TITLE
Store server's data schema

### PR DESCRIPTION
**Pull request by Behzad-rabiei**

### CODE REVIEW

File: __tests__/unit/models/heatMap.model.test.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { HeatMap } from '../../../src/models';
+import { IHeatMap } from '../../../src/interfaces/HeatMap.interface'
+import moment from "moment";
+
```
Comment (mehrdadmms) [inline __tests__/unit/models/heatMap.model.test.ts:4]: Why did you import moment ?
Please remove it if you don't use it.
Btw, add a rule to eslinter to identify unused imports

File: __tests__/unit/models/rawInfo.model.test.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { RawInfo } from '../../../src/models';
+import { IRawInfo } from '../../../src/interfaces/RawInfo.interface'
+import moment from "moment";
```
Comment (mehrdadmms) [inline __tests__/unit/models/rawInfo.model.test.ts:3]:
```suggestion

```

File: src/index.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { databaseService } from './service';
+
+
+
+
+databaseService.connectToDB('salam')
```
Comment (mehrdadmms) [inline src/index.ts:6]:
```suggestion
```
I think this was for development test and should've been removed.

File: src/service/database.service.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    mongoose.set("strictQuery", false);
+    mongoose.connect(config.mongoose.url, { dbName: guildname });
+    // NOTE: connect to a database will not create a new database so for creating database we need crated document in that.
+    // TODO: need
+    if (!await User.findOne({ discordId: 'discordId' })) {
+        await User.create({ discordId: 'discordId' });
+    }
+}
+
+
```
Comment (mehrdadmms) [inline src/service/database.service.ts:20]: I'm not sure but I think this doesn't satisfy the requirement.
Based on issue definition we should be able to create a database for each discord server.
Using `connectToDB` we can connect to/create a database for a server but how can we switch from one db to another if we want to fetch the information from different servers ? 
Should we disconnect and reconnect again ?

**Review by mehrdadmms — CHANGES_REQUESTED** [review]
The only critical comment is the one regarding connecting to multiple databases. 
Other than that one the rest should be addressed but they're not critical

File: __tests__/unit/models/heatMap.model.test.ts (hunk 1 above)
Comment (Behzad-rabiei) [reply __tests__/unit/models/heatMap.model.test.ts:4]: it was a mistake. 
i will add eslint.

**Review by mehrdadmms — CHANGES_REQUESTED** [review]
Add lib to .gitignore

**Review by mehrdadmms — APPROVED** [review]

**Pull request merged**